src/LibraryViewExtension/LibraryViewController.cs

```diff
@@ -190,7 +190,7 @@ internal LibraryView AddLibraryView()
             var browser = view.Browser;
             this.browser = browser;
             sidebarGrid.Children.Add(view);
-            browser.RegisterJsObject("controller", this);
+            browser.RegisterAsyncJsObject("controller", this);
```

Comment:
what if anyother software uses a different version of cefsharp that requires `RegisterJsObject`? I believe it should not be the case. Is there a possible way to check if this object is registered and `window["controller` will return the right object?

Reply:
these functions return void on the c# side so we can't check anything there. We could check in javascript but that wouldn't tell us much more than we knew before as we'll just avoid an exception only to have no functionality, I'd rather just see the exception.

Calling RegisterJsObject again will throw an exception in CEF... :(

 they are supported in version 51 - 63 according to the api docs... which only go back to version 51.

http://cefsharp.github.io/api/63.0.0/html/M_CefSharp_Wpf_ChromiumWebBrowser_RegisterAsyncJsObject.htm

Reply:
yeah I understand that part..is it possible to add exception in JS? like if `window["controller`] is null, then throw (ex)..some clear message ?